composables/useAuth.js

```
@@ -0,0 +1,863 @@
+        photo: `https://api.dicebear.com/7.x/avataaars/svg?seed=${userData.prenom}&backgroundColor=c6f6d5`,
+        created_at: new Date(),
+        bio: "",
+        location: ""
+      }
+
+      // Ajouter à notre "base de données" simulée
+      MOCK_USERS.push(newUser)
+      
+      // Connecter automatiquement
+      const userToReturn = { ...newUser }
+      delete userToReturn.password // Ne pas retourner le mot de passe
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)